packages/orchestrator/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["lib", "../utils/dist", "../types/dist", "../scheduler/dist", "../../.env"],
+    "ext": "js,ts,json",
+    "ignore": ["lib/**/*.test.ts"],
+    "exec": "tsc && tsx -r dotenv/config lib/app.ts dotenv_config_path=./../../.env"
```

Comment:
I get it's handy to have a portable execution, but it's twice the compile when running `npm run dw` (it needs to load the whole project twice)